### ISSUE TITLE
As a User I would like to be able to navigate the site via keyboard

### ISSUE THREAD

**Issue by CGillen** [uoregon-libraries/UO-Library-Omeka-S-Theme#1]
The default theme may or may not supply sufficient keyboard accessibility. An audit should be performed to identify trouble areas for accessibility. Tickets should be created based on the audit and work done to improve keyboard and general accessibility.

To pass this story:
- [x] A user must be able to reasonably access the site with just a keyboard.
- [x] Accessibility should move closer to WCAG 2.1 AA guidelines.

**Comment by CGillen** [comment]
Fixed by #7 
And on-going
